core/vm/logger.go

```diff
@@ -208,6 +208,9 @@ func (l *StructLogger) CaptureStart(env *EVM, from common.Address, to common.Add
 
 	l.statesAffected[from] = struct{}{}
 	l.statesAffected[to] = struct{}{}
+	for _, auth := range authorizationResults {
```

Comment:
Do we need to consider `bytecodes` and `createdAccounts` here?

Reply:
interesting. I didn't find any places where `bytecodes` is changed in `StructLogger`.

Reply:
added in https://github.com/scroll-tech/go-ethereum/pull/1112/commits/eeafee422b8f158d214788b5a6c6f127f8fda60f.

Reply:
`bytecodes` is tracked here: https://github.com/scroll-tech/go-ethereum/blob/eeafee422b8f158d214788b5a6c6f127f8fda60f/core/vm/logger.go#L206